delfin/drivers/fujitsu/eternus/eternus_stor.py

```diff
@@ -352,3 +391,548 @@ def parse_alert(context, alert):
     @staticmethod
     def get_access_url():
         return 'https://{ip}'
+
+    def list_storage_host_initiators(self, ctx):
+        initiator_list = []
+        host_status = self.get_host_status()
+        host_fc_list = self.get_data(consts.GET_HOST_WWN_NAMES,
+                                     consts.HOST_TOTAL)
+        for host_fc in (host_fc_list or []):
+            if len(host_fc) < consts.HOST_TOTAL:
```

Comment:
The host_fc_list returned by self.get_data already contains an empty list. You do not need to add [].
![image](https://user-images.githubusercontent.com/59107077/153787617-df0ccff8-a2d5-4125-9fc4-9887af5874e1.png)

Reply:
Considering later equipment compatibility, module independence is considered.

Reply:
The get_data function is designed as a small independent module, which is more concerned with late maintenance and distrust between different modules

Reply:
Has been modified